### PR TITLE
Add IterationTimer callback

### PR DESCRIPTION
**Pull request by vinnamkim**
### Summary

- We need this callback to compare the model performance migrated from V1.
- Add missing `car_bug_tree` test asset as well.

### How to test
I updated an unit test for this change.

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

**Review by sungmanc — DISMISSED** [review]

File: src/otx/config/train.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 defaults:
   - base_config
   - base: default
+  - callbacks: iteration_timer
```
Comment (harimkang) [inline src/otx/config/train.yaml:4]: One question here, we'll often need to include multiple callbacks, so how do we do that?

**Review by wonjuleee — APPROVED** [review]
LGTM

File: src/otx/config/train.yaml (hunk 1 above)
Comment (vinnamkim) [reply src/otx/config/train.yaml:4]: This file setups four callbacks.
https://github.com/openvinotoolkit/training_extensions/blob/532a020d2af45290f2fea3ac3be521e373485b81/src/otx/config/callbacks/default.yaml#L2-L5

Comment (harimkang) [reply src/otx/config/train.yaml:4]: If so, shouldn't we leave this as the `default`?

**Review by jaegukhyun — APPROVED** [review]

File: src/otx/config/train.yaml (hunk 1 above)
Comment (vinnamkim) [reply src/otx/config/train.yaml:4]: To enable it, there are some more works to be resolved (fixing a proper variable interpolation path).

**Review by harimkang — APPROVED** [review]

**Pull request merged**